pyServer/manifests/linux/monitor-mgmt

```diff
@@ -1,39 +1,49 @@
-echo,#Microsoft Monitoring Agent for Linux Log File
-copy,/var/opt/microsoft/omsagent/*/log/omsagent.log
+echo,# OMS Agent for Linux Log File
+copy,/var/opt/microsoft/omsagent/*/log/omsagent*
 
-echo,#Microsoft Monitoring Agent Configuration Log File
+echo,# DSC Log File
 copy,/var/opt/microsoft/omsconfig/omsconfig.log
 copy,/var/opt/microsoft/omsconfig/omsconfigdetailed.log
 
-echo,#Microsoft Monitoring Agent Configuration File
-copy,/etc/opt/microsoft/omsagent/conf/omsagent.conf
+echo,# OMS Agent Configuration File
+copy,/etc/opt/microsoft/omsagent/*/conf/*.conf
 copy,/etc/opt/microsoft/omsagent/*/conf/omsagent.d/*.conf
 
-echo,#Microsoft Monitoring Agent Agent ID File
```

Comment:
no longer needed, agent id is contained in `/etc/opt/microsoft/omsagent/*/conf/omsadmin.conf` which will be matched by the rule two above